src/plugins/intel_gpu/include/intel_gpu/primitives/crop.hpp

```diff
@@ -91,10 +91,30 @@ struct crop : public primitive_base<crop> {
          const padding& output_padding = padding())
         : primitive_base(id, {input}, output_padding), reference_input(xy_borders.negate()), offsets(xy_borders) {}
 
+    /// @brief Constructs crop primitive.
+    /// @param id This primitive id.
+    /// @param inputs Input primitive id vector.
+    /// @param reference_input Reference input tensor with the required dimensions.
+    /// @param offsets Input offsets.
+    /// @param output_idx Output data index of splited output.
+    crop(const primitive_id& id,
+         const std::vector<primitive_id>& inputs,
+         const tensor& reference_input,
+         const tensor& offsets,
+         const int output_idx,
+         const size_t num_splits = 0,
```

Comment:
isnt it better to set the default value as 1?

Reply:
Done